src/Containers/User/UserContactEdit.js

```diff
@@ -0,0 +1,64 @@
+import userEvent from "@testing-library/user-event";
+import React, { useState } from "react";
+import { useParams, useNavigate, useLocation } from "react-router-dom";
+
+
+export default function UserContactEdit() {
+  const { uid } = useParams();
+  const navigate = useNavigate();
+  const location = useLocation();
+  const [email, setEmail] = useState(location.state.email);
+  const [phone, setPhone] = useState(location.state.phone);
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    const emailData = { address : email,
+                        emailType : location.state.emailType}
+    const phoneData = { number : phone,
+                        phoneType : location.state.phoneType}
+    fetch(`https://gy8a0m85ci.execute-api.us-east-1.amazonaws.com/test/user/${uid}/contact/email/${location.state.emailId}`, {
+      method: 'PUT',
+      headers: {
+        'Content-type': 'application/json',
+      },
+      body: JSON.stringify(emailData),
+    })
```

Comment:
catch errors, and create a component such as `NoticeBanner` to pass in the errors, so that you can reuse this notice banner to indicate success or failure and all pages can reuse this banner component.